docs/gpt.rst

```diff
@@ -0,0 +1,67 @@
+gpt
+=======
+
+.. dfhack-tool::
+    :summary: AI-generated written content!
+    :tags: fort gameplay
+
+Enables a UI for submitting knowledge item descriptions to OpenAI for generating
+poetry, star charts, and excerpts from longer works such as biographies, dictionaries,
+treatises on technological evolution, comparative biographies, cultural histories,
+autobiographies, cultural comparisons, essays, guides, manuals, and more.
+
+``enable gpt``
+=======
+Enables the plugin. The overlay will be shown when a knowledge item or unit view sheet is open.
+
+``disable gpt``
+=======
+Disables the plugin.
```

Comment:
this actually isn't necessary. players can turn the overlay on and off in `gui/control-panel`.